exasol/ds/sandbox/cli/commands/create_docker_image.py

```diff
@@ -27,18 +32,26 @@
 def create_docker_image(
         repository: str,
         version: str,
-        publish: bool,
+        registry_user: str,
+        registry_password: str,
         keep_container: bool,
         log_level: str,
 ):
     """
-    Create a Docker image for data-science-sandbox and deploy
-    it to a Docker repository.
+    Create a Docker image for data-science-sandbox.  If username and password
+    for the Docker registry are specified then deploy the image to the registry.
     """
     set_log_level(log_level)
-    DssDockerImage(
+    creator = DssDockerImage(
         repository=repository,
         version=version,
-        publish=publish,
         keep_container=keep_container,
-    ).create()
+    )
+    if registry_user and registry_password:
+        creator.registry = DockerRegistry(
+            creator.repository,
+            registry_user,
+            registry_password,
+        )
+    print(f'user {creator.registry.username} password {registry_password}')
```

Comment:
Please try to avoid printing the password to logs or any other output.
You can consider introducing a small `Credentails` wrapper grouping username and password, which automatically hides/replace passwords with `***` when printed.